src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('.field-text');
+const parents = document.querySelectorAll('.field');
+
+for (let input = 0; input < inputs.length; input++) {
```

Comment:
```suggestion
for (let i = 0; i < inputs.length; i++) {
```

Using `input` as the name for the counter can be slightly confusing. Better use `i`, and in case of nested loops you can use for example `j`/`k`